browser_env/actions.py

```diff
@@ -308,6 +322,7 @@ class ActionTypes(IntEnum):
     CHECK = 15
     SELECT_OPTION = 16
 
+    CLEAR = 16
```

Comment:
We need to give CLEAR a unique ID, what about using 18 instead? 16 is the SELECT_OPTION action.

Reply:
just changed it to 18